lib/livebook_web/live/apps_live.ex

```diff
@@ -31,32 +31,45 @@ defmodule LivebookWeb.AppsLive do
       current_user={@current_user}
       saved_hubs={@saved_hubs}
     >
-      <div class="p-4 md:px-12 md:py-7 max-w-screen-lg mx-auto space-y-4">
-        <div>
-          <LayoutHelpers.title text="Apps" />
-          <p class="mt-4 mb-8 text-gray-700">
-            <%= if @sessions == [] do %>
-              No apps currently running.
-            <% else %>
-              These apps are currently running.
-            <% end %>
-          </p>
+      <div class="p-4 md:px-12 md:py-7 max-w-screen-lg mx-auto">
+        <LayoutHelpers.title text="Apps" />
+        <.app_list sessions={@sessions} />
+      </div>
+    </LayoutHelpers.layout>
+    """
+  end
+
+  defp app_list(%{sessions: []} = assigns) do
+    ~H"""
+    <div class="mt-8 p-5 flex space-x-4 items-center border border-gray-200 rounded-lg">
+      <div>
+        <.remix_icon icon="windy-line" class="text-gray-400 text-xl" />
+      </div>
+      <div class="grow flex items-center justify-between">
+        <div class="text-gray-600">
+          You do not have any apps running. <br /> You can deploy new apps by clicking
+          <.remix_icon icon="rocket-line" class="align-sub text-lg" /> in the session sidebar.
```

Comment:
```suggestion
          <.remix_icon icon="rocket-line" class="align-sub text-lg" /> in the notebook sidebar.
```

I think session may not be immediately clear and "notebook session sidebar" is too long. Perhaps we can break it into more steps: "You can deploy new apps by opening a notebook and clicking [R] in the sidebar".

Reply:
I'd rather avoid calling the session page notebook for consistency. THe other version sounds good to me!